Appwrite/Appwrite_User_Authentication.ipynb

```diff
@@ -131,18 +148,44 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 28,
    "id": "69019e17-53eb-434a-8837-ab33b6a5d0ce",
    "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-10-12T14:14:15.011512Z",
+     "iopub.status.busy": "2023-10-12T14:14:15.011285Z",
+     "iopub.status.idle": "2023-10-12T14:14:16.013409Z",
+     "shell.execute_reply": "2023-10-12T14:14:16.012814Z",
+     "shell.execute_reply.started": "2023-10-12T14:14:15.011490Z"
+    },
     "papermill": {},
     "tags": []
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "👌 Well done! Your Secret has been sent to production. \n",
+      "\n",
+      "PS: to remove the \"Secret\" feature, just replace .add by .delete\n",
+      "👌 Well done! Your Secret has been sent to production. \n",
+      "\n",
+      "PS: to remove the \"Secret\" feature, just replace .add by .delete\n",
+      "👌 Well done! Your Secret has been sent to production. \n",
+      "\n",
+      "PS: to remove the \"Secret\" feature, just replace .add by .delete\n"
+     ]
+    }
+   ],
    "source": [
-    "endpoint = \"https://cloud.appwrite.io/v1\" #replace with your end-point here.\n",
-    "project = \"6526da963cd09677a562\" #replace with your project here.\n",
-    "key = \"04e727e29a0d284ef05a9206ab13d288fff61f96193962fe3c9238c116cdd935e77c4dbecf6ef16d3d485a1d48c202b4cad3b16a4dd30847e218b6ebbfec341ab62fdacd29616030ce16b8093c677e1f615f59588a1b58e32d488e60b6b04736f792f6b3e8cbe2e03bf447245a87913f38aac111a7b53cf59c68831e02d732d8\"\n",
-    "#replace key with your key."
+    "# replace endpoint , project , key with your credentials.\n",
+    "naas.secret.add(name=\"endpoint\",secret=\"https://cloud.appwrite.io/v1\")\n",
```

Comment:
@MSaiKiran9 You do not need to add the API key itself.
![image](https://github.com/jupyter-naas/awesome-notebooks/assets/81156510/f1e0d3ba-5257-47cc-8070-f1d703eb778d)

Just this will suffice the requirement. You can take a look at previous notebooks to get an idea about the same.